<launch>
    <node pkg="au_vision" type="detector_server" name="NaiveBuoyDetector" output="screen">
        <param name="detector" value="naive_buoy_detector"/>
    </node>
</launch>